### PR TITLE
fix(indexer): Increase resources for flaky indexer tests

### PR DESCRIPTION
**Pull request by roninjin10**
- indexer tests are flaky from 10 minute timeouts increase resources to xlarge

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #6810** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6810" target="_blank">Graphite</a> 
    * **PR #6808** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6808" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6808?utm_source=stack-comment).

**Review by hamdiallam — APPROVED** [review]

**Comment by trianglesphere** [comment]
You also need to increase the parallel flag to 8, but tbh I think it is an issue with the tests b/c when the tests pass, it appears to just take 2-3 minutes

File: .circleci/config.yml [hunk 1]
```diff
@@ -897,7 +897,7 @@ jobs:
     docker:
       - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:latest
       - image: cimg/postgres:14.1
-    resource_class: large
+    resource_class: xlarge
     steps:
       - checkout
       - check-changed:
```
Comment (trianglesphere) [inline .circleci/config.yml:903]: Outside where I can comment, but please also remove the `--timeout 2m` from the lint step.

**Review by mslipper — CHANGES_REQUESTED** [review]
Do we absolutely have to increase executor size? [CPU usage](https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/24287/workflows/478527aa-0c5d-470e-b91b-6d17ac893411/jobs/1092522/resources) suggests that we are only 25% utilized for most of the build. The xlarge executor is 2x more expensive than the large, and those costs really add up.

**Comment by roninjin10** [comment]
closing in favor of debugging issue

**Comment by roninjin10** [comment]
@trianglesphere your comment is here https://github.com/ethereum-optimism/optimism/pull/6810

**Pull request closed without merging**